### DISCUSSION TITLE
How can I import exceljs for use in the browser (react-app)?

### DISCUSSION THREAD

**Discussion by dimitrisdovinos** [exceljs/exceljs#2472] [Q&A]
I am trying to import exceljs in order to export data to xlsx. This is within the context of a react application and it should, therefore, work in the browser. I have tried several ways of importing the library but I am having no luck. Is there something particularly tricky that I am missing?
Here is are some of the imports I tried:

> import Excel from 'exceljs/dist/es5/exceljs.browser';
> import Excel from 'exceljs';

Thanks in advance

**Comment by SyedAsimAliSE** [comment]
in case any one still stuck here try to import it like this : ```import * as Excel from "exceljs";```
